jazz_asset-event-handler/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "jazz_asset-event-handler",
+    "version": "0.0.1",
+    "description": "The event handler funtion to handle asset related events and catalog updates",
+    "main": "index.js",
+    "private": true,
+    "author": [],
+    "scripts": {
+        "test": "mocha"
+    },
+    "dependencies": {
+        "string-template": "*",
+        "json-query": "*",
+        "lodash": "*",
```

Comment:
if the use is only for _includes then i can update the code with ES5 native and remove this totally.